packages/lit-element/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "lit-element",
+  "version": "3.0.0-pre.0",
+  "description": "A simple base class for creating fast, lightweight web components",
+  "license": "BSD-3-Clause",
+  "homepage": "https://lit-element.polymer-project.org/",
+  "repository": "Polymer/lit-element",
+  "type": "module",
+  "main": "index.js",
+  "module": "index.js",
+  "typings": "index.d.ts",
+  "directories": {
+    "test": "test"
+  },
+  "files": [
+    "/lib/",
+    "/src/",
+    "!/src/demo/",
+    "!/src/test/",
+    "/lit-element.d.ts",
+    "/lit-element.d.ts.map",
+    "/lit-element.js",
+    "/lit-element.js.map",
+    "/ts3.4/"
+  ],
+  "scripts": {
+    "build": "tsc",
+    "build:watch": "tsc --watch",
+    "build:babel-test": "babel src/test/lib/decorators_test.ts --out-file test/lib/decorators-babel_test.js",
+    "test": "wtr test/*test.js --node-resolve --root-dir ../../",
+    "test:watch": "wtr test/*test.js --node-resolve --root-dir ../../ --watch",
+    "checksize": "rollup -c ; cat lit-element.bundled.js | gzip -9 | wc -c ; rm lit-element.bundled.js",
+    "format": "prettier src/* --write",
+    "lint": "tslint --project ./",
+    "prepublishOnly": "node check-version-tracker.cjs && npm run lint && npm test",
+    "prepare": "npm run build",
```

Comment:
CI failed due to monorepo lerna bootstrap failing, due to lit-element's npm install running first, causing this prepare to run, before lit-html had been built, thus can't find lit-html module.

Seems like for now we should remove `prepare` on individual packages? And maybe add a monorepo-level prepare?

Reply:
Done.